.github/actions/install-mvnd/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'install-mvnd'
+description: 'Install the maven daemon'
+inputs:
+  version:
+    description: 'The version of the maven daemon to install'
+    required: true
+    default: '0.7.1'
+outputs:
+  mvnd-dir:
+    description: "The directory where the command mvnd is located"
+    value: ${{ steps.mvnd-location.outputs.mvnd-dir }}
+runs:
+  using: "composite"
+  steps:
+    - run: curl -fsSL -o mvnd.zip https://github.com/apache/maven-mvnd/releases/download/${{ inputs.version }}/mvnd-${{ inputs.version }}-linux-amd64.zip
+      shell: bash
```

Comment:
I am not entirely sure if we are allowed to download and run code freely on the GH actions (any idea @oscerd?), but I think that we need to include the following here:
1. Cache the hash sum and verify it before running.

Reply:
You mean that you would like to add a checksum to guarantee the consistency of the file?

Reply:
Yes. Unfortunately it's not signed, so it's the best we can do.

Reply:
@orpiske checker added, hoping that it was what you had in mind